packages/react/src/toggle-group/ToggleGroup.tsx

```diff
@@ -0,0 +1,20 @@
+import * as RadixToggleGroup from "@radix-ui/react-toggle-group";
+import { forwardRef } from "react";
+
+import { Box, type BoxProps } from "../box";
+
+type ToggleGroupProps = Omit<BoxProps<typeof RadixToggleGroup.Root>, "type">;
+
+export const ToggleGroup = forwardRef<HTMLDivElement, ToggleGroupProps>(
+  ({ children, ...props }, ref) => {
+    return (
+      <Box asChild {...props}>
```

Comment:
i think we should use `ButtonGroup` here - or at the very least a Flex

Reply:
Using `ButtonGroup` here with 0 gap will give a line separating the buttons. Using `Flex` for this bhai